proposed/phpdoc-tags.md

```diff
@@ -917,9 +892,9 @@ Each Constant or Property definition or Variable where the type is ambiguous
 or unknown SHOULD be preceded by a DocBlock containing the @var tag. Any
 other variable MAY be preceded with a DocBlock containing the @var tag.
 
-The @var tag MUST contain the name of the element it documents. An exception
-to this is when property declarations only refer to a single property. In this
-case the name of the property MAY be omitted.
+The @var tag MUST contain the name of the element it documents, unless this
+property declaration only refers to a single property. In this case the name of
+the property MAY be omitted.
```

Comment:
Maybe

```suggestion
the property SHOULD be omitted.
```

Writing the property name is quite strange to be honest.

~~~php
/** @var string $name */
private $name;
~~~

Reply:
Also do we intentionally disallow ommiting element name when using it with variable assignment?

```php
/** @var string */
$name = 'John';
```

Reply:
Regarding the property definition, this is written this way because of multi properties defenition in a single line.
When you add the property name the @var will only be applied to that property. Otherwise it will be applied to all properties.

But I agree it needs some clarification.